packages/wdio-shared-store-service/src/server.js

```diff
@@ -0,0 +1,50 @@
+const polka = require('polka')
+const { json } = require('@polka/parse')
+
+const store = {}
+
+const app = polka()
+    /**
+     * middleware
+     * `json` middleware transforms body to json for every request or returns empty object
+     */
+    .use(json(), validateBody)
+
+    // routes
+    .post('/get', (req, res) => {
+        res.end(JSON.stringify({ value: store[req.body.key] }))
+    })
+    .post('/set', (req, res) => {
+        store[req.body.key] = req.body.value
+        return res.end()
+    })
+
+const startServer = async () => new Promise((resolve, reject) => {
+    /**
+     * run server on a random port, `0` stands for random port
+     * > If port is omitted or is 0, the operating system will assign
+     * > an arbitrary unused port, which can be retrieved by using `server.address().port`
+     * https://nodejs.org/api/net.html#net_server_listen_port_host_backlog_callback
+     */
+    app.listen(0, (err) => {
```

Comment:
Small question, does this call occurs before selenium server started(I mean e.g. selenium-standalone service)?
I am worried about very rare case when random server port by coincidence will be `4444` when usually selenium server will start.

Reply:
It starts same time all the services are started (in parallel)

Reply:
Before proceeding I'll check how it works on windows as well

Reply:
The ranges should be between 49152 and 65535

@BorisOsipov

Reply:
@mgrybyk nice! thanks for research.